gfx/font.h

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2022 Robin Lindén <dev@robinlinden.eu>
+//
+// SPDX-License-Identifier: BSD-2-Clause
+
+#ifndef GFX_FONT_H_
+#define GFX_FONT_H_
+
+#include <string_view>
+
+namespace gfx {
+
+struct Font {
+    std::string_view font;
+};
+
+struct FontSize {
+    int px{10};
+};
+
+} // namespace gfx
+
+#endif
```

Comment:
Missing newline at eof. I thought this was enforced by clang-format, but I guess not.. Also missing in painter.{cpp,h}